src/Breadcrumb/index.jsx

```diff
@@ -57,6 +59,7 @@ Breadcrumb.propTypes = {
 
 Breadcrumb.defaultProps = {
   activeLabel: undefined,
+  ariaLabel: undefined,
```

Comment:
nit: should we use "breadcrumb" value as default instead of `undefined`?

Reply:
Thanks, fixed